src/renderer/viz/expressions/linear.js

```diff
@@ -3,7 +3,8 @@ import { checkExpression, implicitCast, checkType, checkMaxArguments, clamp } fr
 import { globalMin, globalMax } from '../expressions';
 import { castTimeRange, msToDate } from '../../../utils/util';
 import IdentityCodec from '../../../codecs/Identity';
-import TZDate from '../../../utils/time/TZDate';
```

Comment:
I find the new name much clearer, but I guess that we should consider this a breaking change

Reply:
It's not an expression, why should it be then a breaking change?

Reply:
All right then, I thought it was exposed somehow.